deployment/helm-templates/values.yaml.prod.template

```diff
@@ -56,15 +55,15 @@ ckan:
     resources:
       limits:
         cpu: 800m
-        memory: 4G
```

Comment:
The only thing that seems off here is that I think these should remain as they were (unless the client requested lower resources).